docs/source/install/upgrades.rst

```diff
@@ -170,10 +170,10 @@ any skipped versions:
 v3.4
 ''''
 
-*  |st2| now uses python 3 on RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
+*  |st2| now uses python 3 on Ubuntu 16 and RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
```

Comment:
@amanda11 Do you refer to this comment about packs migration or any other place?
I believe it applies for both U16 and EL7 now.

Reply:
Upgrade-notes.rat file, was what I was referring to.

Reply:
Thanks, I missed that!

Added in https://github.com/StackStorm/st2docs/pull/1047/commits/8ac1f0af4db8e4d4b141fb77e384bd856c756617, reworded it a little bit to accommodate the python 2 removal.